controllers/apply/under10k/form.js

```diff
@@ -1337,12 +1381,28 @@ module.exports = function ({
 
         const enriched = clone(data);
 
-        enriched.projectStartDate = dateFormat(enriched.projectStartDate);
+        function normaliseProjectStartDate() {
+            /**
+             * If projectStartDateCheck is `asap` then pre-fill
+             * the projectStartDate to today
+             */
+            if (
+                flags.enableNewCOVID19Flow &&
+                get('projectStartDateCheck')(data) === 'asap'
```

Comment:
Ended up going with the most direct check I could think of in the `forSalesForce` method. Rather than trying to hope that `joi` has pre-filled a value for us, I'm instead doing a direct check for `asap` as the date option and setting the start date to the current date.